src/main/java/nc/students/ayaz/model/Comment.java

```diff
@@ -1,13 +1,24 @@
-package nc.students.ayaz;
+package nc.students.ayaz.model;
 
 import cobertura.IgnoreDuringCodeCoverage;
 
-class Comment {
+public class Comment {
 
     private String author;
+
     private String text;
 
-    Comment(String author, String text) {
+    @IgnoreDuringCodeCoverage
+    public String getAuthor() {
+        return author;
+    }
+
+    @IgnoreDuringCodeCoverage
+    public String getText() {
+        return text;
+    }
+
+    public Comment(String author, String text) {
```

Comment:
&#x1F34E; **Issue**
Also, remember how you should structure components of your class. Instance methods shouldn't go before constructors.